File: app/src/main/java/com/platdmit/simplecloudmanager/domain/repo/UpdateScheduleRepo.java
package com.platdmit.simplecloudmanager.domain.repo;

import com.platdmit.simplecloudmanager.data.database.entity.DbUpdateSchedule;

import java.util.List;

public interface UpdateScheduleRepo {
    List<DbUpdateSchedule> getUpdateTimeList();
    long getUpdateTime(String key);
    void setUpdateTime(String key, long nextUpdate);
}

File: app/src/main/java/com/platdmit/simplecloudmanager/domain/repo/UpdateScheduleRepo.kt
package com.platdmit.simplecloudmanager.domain.repo

import com.platdmit.data.database.entity.DbUpdateSchedule

interface UpdateScheduleRepo {
    fun getUpdateTimeList(): List<DbUpdateSchedule>?
    fun getUpdateTime(key: String): Long
    fun setUpdateTime(key: String, nextUpdate: Long)
}